src/debug/wallLabelsDebug.ts

```
@@ -0,0 +1,294 @@
+  clearLabels(scene: THREE.Scene): void {
+    this.labels.forEach(label => {
+      scene.remove(label)
+
+      // Dispose of geometries and materials
+      if ('geometry' in label) {
+        (label.geometry as THREE.BufferGeometry).dispose()
+      }
+      if ('material' in label) {
+        const material = label.material
+        if (Array.isArray(material)) {
+          material.forEach(mat => mat.dispose())
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)